widgets/blueprints/src/types.ts

```diff
@@ -8,6 +8,8 @@ export interface BlueprintsWidgetConfiguration {
     hideFailedBlueprints: boolean;
     showComposerOptions: boolean;
     marketplaceTabs: Stage.Common.BlueprintMarketplace.Tab[];
+    marketplaceDisplayStyle: 'table' | 'catalog';
```

Comment:
Maybe we can have something like
```suggestion
    marketplaceDisplayStyle: Stage.Common.BlueprintMarketplace.DisplayStyle;
```
?
(see my comment about extracting a type for `'table' | 'catalog'`)